test/cmdlineTests/linking_strict_assembly/output

```diff
@@ -10,9 +10,6 @@ object "a" {
 
 
 Binary representation:
-73123456789012345678901234567890123456789050
+
 
 Text representation:
-  linkerSymbol("f919ba91ac99f96129544b80b9516b27a80e376b9dc693819d0b18b7e0395612")
```

Comment:
Why was this bytecode and the ones in a few other tests removed?

Reply:
Because the optimiser now is turned on and these are side effect free statements.

Reply:
Ok. Then I agree with @chriseth that these tests should be adjusted so they don't lose their test meaning

Reply:
Well, that's what the first commit is doing. And also this, but nobody responds: https://github.com/ethereum/solidity/pull/10286#discussion_r541063688